python/cuspatial/benchmarks/io/bench_geoseries.py

```diff
@@ -0,0 +1,21 @@
+# Copyright (c) 2022, NVIDIA CORPORATION.
+
+"""Benchmarks of GeoSeries methods."""
+
+from config import cuspatial
+
+
+def bench_from_geoseries_100(benchmark, gpdf_100):
```

Comment:
This PR seems missing a few fixtures?

Reply:
Oh strange, they should have been in `conftest.py`

Reply:
Do you mean a separate `conftest` for benchmark or the one existing for tests? I don't see the former included in this PR and I don't see `gpdf_100` in `tests/conftest.py`.

Although you may have a conftest shared by both tests and benchmarks, at this point I think it's best to just keep them separate.

Reply:
I found the conftest.py specialized for benchmarks and added it here.